Formula/vivid.rb

```diff
@@ -0,0 +1,29 @@
+class Vivid < Formula
+  desc "Generator for LS_COLORS with support for multiple color themes"
+  homepage "https://github.com/sharkdp/vivid"
+  url "https://github.com/sharkdp/vivid/archive/v0.5.0.tar.gz"
+  sha256 "d4864169dca54e44c1f3a6f09bcf5048190e6a97ee0a9887437dbc7f5ed6aaaa"
+
+  depends_on "rust" => :build
+
+  def install
+    ENV.append_to_cflags "-fno-stack-check" if DevelopmentTools.clang_build_version >= 1010
+    system "cargo", "install", "--locked", "--root", prefix, "--path", "."
+    # Update when https://github.com/sharkdp/vivid/blob/master/ci/before_deploy.bash changes.
+    pkgshare.install "config/filetypes.yml", "themes"
+  end
+
+  # Hardcoded to /usr/share/vivid or $HOME/.config/vivid, doesn't check XDG_CONFIG_HOME.
```

Comment:
Please make a pull request upstream to fix this instead.

Reply:
Sounds good, marking this as a draft until that is done (https://github.com/sharkdp/vivid/issues/25#issuecomment-642570464)